etcdctl/ctlv3/ctl.go

```diff
@@ -114,11 +114,10 @@ func Start() error {
 
 func MustStart() {
 	if err := Start(); err != nil {
-		if rootCmd.SilenceErrors {
-			cobrautl.ExitWithError(cobrautl.ExitError, err)
-		} else {
+		if !rootCmd.SilenceErrors {
 			os.Exit(cobrautl.ExitError)
 		}
+		cobrautl.ExitWithError(cobrautl.ExitError, err)
```

Comment:
I think this can also be simplified as just removing the else condition, as [`cobrautl.ExitWithError()`](https://github.com/etcd-io/etcd/blob/507c0de87bd5034e3de4ab76ebf96b54dae0cd52/pkg/cobrautl/error.go#L37-L40) already does an `os.Exit()`.

```suggestion
		if rootCmd.SilenceErrors {
			cobrautl.ExitWithError(cobrautl.ExitError, err)
        }
		os.Exit(cobrautl.ExitError)
```